app/assets/javascripts/discourse/app/components/toolbar-popup-menu-options.gjs

```diff
@@ -32,7 +32,9 @@ export default class ToolbarPopupmenuOptions extends Component {
     if (content.condition) {
       let label;
       if (content.label) {
-        label = i18n(content.label);
+        label = content.labelArgs
+          ? i18n(content.label, content.labelArgs)
+          : i18n(content.label);
```

Comment:
I wonder if it wouldn't be simpler to use `DButton's` `label` vs `translatedLabel`, and in this case pass `translatedLabel` instead of `label`. wdyt?

Reply:
We can't use `@label` for `DButton` here because we need to append the shortcut onto the end of the label. However yes I think we can support sending through `translatedLabel` here instead of `labelArgs`, will change that